src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -180,4 +185,55 @@ public void mercury_search_redirectToNewResultsPageFromNoResults() {
     Assertion.assertFalse(searchResults.isNoResultsPagePresent());
     Assertion.assertTrue(searchResults.areResultsPresent());
   }
+
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_defaultResultsNumberOnSearchResultsPage")
+  public void mercury_search_defaultResultsNumberOnSearchResultsPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)